### DISCUSSION TITLE
Does anyone know where I can find this node?

### DISCUSSION THREAD

**Discussion by ycyy** [comfyanonymous/ComfyUI#4092] [Q&A] [answered]
![PixPin_2024-07-24_10-47-55](https://github.com/user-attachments/assets/020035bd-048a-4392-bc92-0f1de45a0ffe)

**Comment by ltdrdata** [comment, marked as answer]
That is just a normal CLIPTextEncode node.
Just you need to convert 'text' widget to input on contextmenu.

**Reply by ycyy** [reply]
thank you very much,i got it.
